addons/mission/functions/fnc_switchAction.sqf

```diff
@@ -2,19 +2,21 @@
 /*
  * Author: Mike
  * Functionality for toggling switches, when switch is "OFF" the light is red and power levels are 0, when switch is "ON" the light is green and power levels are 1 or player defined.
- *
+ * Designed for use with the "Transfer Switch" object.
  * Default state is 0 or 1
  *
+ * Voltages control the power level indicators on the switch itself. Values range from 0-1.
+ *
  * Switch state can be checked by getVariable
- * My_Switch getVariable ["TAC_switchState", false];
+ * My_Switch getVariable ["tac_mission_switchState", false];
```

Comment:
Should probably just show use with `MQGVAR`?

Reply:
Sure